python/tests/integration/server_manager.py

```diff
@@ -102,7 +131,11 @@ def __init__(self, port, timeout=None, dispatcher=None):
     def serve(self):
         self.tchannel.host(self.dispatcher).listen()
         self.ready = True
-        tornado.ioloop.IOLoop.current().start()
+        try:
+            tornado.ioloop.IOLoop.instance().start()
+            print "stop"
+        except:
+            pass
 
     def shutdown(self):
-        tornado.ioloop.IOLoop.current().stop()
+        tornado.ioloop.IOLoop.instance().stop()
```

Comment:
We should always be using `current` over `instance`, why did you need to change these?